fast/stages/2-networking-d-separate-envs/README.md

```diff
@@ -112,15 +112,12 @@ This configuration is battle-tested, and flexible enough to lend itself to simpl
 ### VPCs
 
 VPCs are defined in separate files, one for each of `prod` and `dev`.
-Each file contains the same resources, described in the following paragraphs.
 
-The **project** ([`project`](../../../modules/project)) contains the VPC, and enables the required APIs and sets itself as a "[host project](https://cloud.google.com/vpc/docs/shared-vpc)".
+These files contain different resources:
```

Comment:
which files?

Seems like this header doesn't match what is enumerated later

Reply:
![image](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/assets/66007/e58903be-3798-43a4-9aa4-3ff63a345efa)

`defined in separate files`...